src/OrchardCore.Modules/OrchardCore.Taxonomies/Drivers/TermPartContentDriver.cs

```diff
@@ -71,11 +71,13 @@ public override IDisplayResult Display(ContentItem model, BuildDisplayContext co
     {
         var query = await _contentsTaxonomyListQueryService.QueryAsync(termPart, pager);
 
+        var totalItems = await query.CountAsync();
```

Comment:
Why does moving this here fix the issue? Or is this only a code style change?

Reply:
because of the bug fixed here https://github.com/sebastienros/yessql/pull/659/files  no need to document it since a YesSQL fix coming soon.

Reply:
I really don't want to add a comment that will become obsolete very quick. I pinged @sebastienros on the YesSQL PR so we should be merging and releasing a patch soon.

Reply:
The code will become obsolete as well, so why change it? :) An update together with the YesSql reference will be necessary here anyway.

Reply:
when we update yessql, we don't need to change the code. it will work correctly either way. so I want to fix the problem on hand on OC without cluttering the code with comments that will only be good for couple of days. once we update yessql, the code will stay the same because it will behave the same way either way